requirements/edx/base.txt

```diff
@@ -6,6 +6,7 @@
 
 beautifulsoup4==4.1.3
 beautifulsoup==3.2.1
+before_after==0.1.3
```

Comment:
@mollydb, I've been informed by this document that I need to notify you of this change for licensing reasons. This library is licensed under GPLv2 (https://github.com/c-oreills/before_after/blob/master/LICENSE), and I checked in the Open Source chat room before starting this effort.

@edx/devops, `support@edx.org` is also mentioned as needs-to-be-notified above. This library can be found at https://github.com/c-oreills/before_after, it's just sugar on top of the already existing mock library.